<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtoFileSaveWidget</class>
 <widget class="QWidget" name="OtoFileSaveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="saveOptionGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="title">
      <string>保存位置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>保存到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="saveToSrcRadioButton">
          <property name="text">
           <string>源文件</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">saveFileNameButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="saveToCustomRadioButton">
          <property name="text">
           <string>指定路径</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">saveFileNameButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="OtoFileNameEditWithSaveBrowse" name="saveFileNameEdit" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QWidget" name="secondFileNameWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <property name="leftMargin">
          <number>12</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="secondFileNameCheckBox">
           <property name="text">
            <string>保存被修改的项到</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="OtoFileNameEditWithSaveBrowse" name="secondFileNameEdit" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OtoFileNameEditWithSaveBrowse</class>
   <extends>QWidget</extends>
   <header>widgets/otofilenameeditwithbrowse.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="saveFileNameButtonGroup"/>
 </buttongroups>
</ui>
